stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/composer/MessageComposerViewModel.kt

```diff
@@ -52,6 +60,18 @@ public class MessageComposerViewModel(
      */
     public val lastActiveAction: Flow<MessageAction?> = messageComposerController.lastActiveAction
 
+    /**
+     * Initializing properties:
+     * - Listening to cooldownTimer value's updates and propagating the update to messageInputState.
```

Comment:
Just for reference - I've exposed the ComposerState in the controller, in anotehr PR:

https://github.com/GetStream/stream-chat-android/blob/66c15fafcd449b0abca4f552d63826e55f8bd626/stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/composer/MessageComposerController.kt